src/test/java/uk/gov/hmcts/cmc/claimstore/services/ServiceAuthProviderServiceTest.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ServiceAuthProviderServiceTest {
```

Comment:
For me personally that test is useless the same way as `ServiceAuthProviderService` class.